docs/validators/key-management/multikey-nodes.md

```diff
@@ -152,3 +152,21 @@ example-0 e296e97524483e6b59...
 example-1 585ddceb6b7bf0d308...
 random-0  791c7e2bd6a5fb1371...
 ```
+
+### Security notes for the multikey nodes
+
+As stated above, the multikey feature is able to use any number of keys with just 4 nodes. 
+At the first sight, this can be seen as a security degradation in terms of means of attacking a large staking provider but there are ways to mitigate these concerns as explained in the following list:
+1. use the recommendation found in this page regarding the maximum number of keys per multikey group;
+2. for each main multikey group use at least one backup multikey group in case something bad happens with the main group;
+3. use the NamedIdentity configuration explained above to hide the BLS keys and their declared identity from the actual nodes that manages the keys.
```

Comment:
NamedIdentity -> \`NamedIdentity\`

and maybe find a replacement for 'hide'.

Reply:
done, replaced `hide` with `obfuscate`